### PR TITLE
Create readme.md

### PR DESCRIPTION
**Pull request by yahyamohmuedpro99**
# Types Conflict Issue - Notice and Solution

### CODE REVIEW

File: subqueries/coreum-testnet/src/readme.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Types Conflict Issue - Notice and Solution
+
+## Issue Description
+When running the indexer, an issue arises due to conflicts in the types of `MsgMint` between FT (Fungible Token) and NFT (Non-Fungible Token) within the `CosmosMessageTypes.ts` file. These message types lack differentiation, causing conflict during code generation or compilation.
```
Comment (metmirr) [inline subqueries/coreum-testnet/src/readme.md]: Please add a complete path name like this `./src/types/CosmosMessagTypes.ts`

File: subqueries/coreum-testnet/src/readme.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  MsgBurn as MsgBurnNFT,
+  MsgFreeze as MsgFreezeNFT
+} from "./proto-interfaces/coreum/asset/nft/v1/tx";
+
+// ... (additional code)
+
+export type MsgIssueClassMessage = CosmosMessage<MsgIssueClass>;
+export type MsgMintMessageNFT = CosmosMessage<MsgMintNFT>;
+export type MsgBurnMessageNFT = CosmosMessage<MsgBurnNFT>;
+export type MsgFreezeMessageNFT = CosmosMessage<MsgFreezeNFT>;
```
Comment (metmirr) [inline subqueries/coreum-testnet/src/readme.md:26]: Add  ``` at the end of the code section to close the section.

**Review by metmirr — CHANGES_REQUESTED** [review]

File: subqueries/coreum-testnet/src/readme.md (hunk 1 above)
Comment (yahyamohmuedpro99) [reply subqueries/coreum-testnet/src/readme.md]: can you check now please

File: subqueries/coreum-testnet/src/readme.md (hunk 2 above)
Comment (yahyamohmuedpro99) [reply subqueries/coreum-testnet/src/readme.md:26]: can you check now please

**Review by metmirr — APPROVED** [review]

**Pull request merged**